Containerfile

```diff
@@ -10,12 +10,15 @@ RUN echo "[git-annex]" >> /etc/yum.repos.d/git-annex.repo \
          && echo "gpgcheck=0" >> /etc/yum.repos.d/git-annex.repo \
          && echo "enabled=1" >> /etc/yum.repos.d/git-annex.repo
 
-RUN dnf -y module enable nodejs:14 ruby:2.7
-RUN dnf -y install nodejs vim git rpmdevtools git-annex-standalone wget python3 ruby jq ruby-devel make gcc-c++ postgresql-devel libvirt-devel libxml2-devel libcurl-devel systemd-devel
+RUN dnf install -y epel-release && \
+    dnf -y install nodejs vim git rpmdevtools git-annex-standalone wget ruby jq ruby-devel make gcc-c++ mock postgresql-devel libxml2-devel libcurl-devel systemd-devel \
+    python3 python3-pip python3-ruamel-yaml python3-requests python3-packaging
 
 RUN npm install npm2rpm --global
 
-RUN pip3 install packaging requests ruamel-yaml
+RUN gem install gem2rpm
+
+RUN pip3 install rpmspectool obal
```

Comment:
Also surprises me `rpmspectool` isn't in EPEL (https://src.fedoraproject.org/rpms/rpmspectool). Almost makes me think people typically use Fedora to develop ;)